app/serializers/education_benefits_claim_serializer.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
-class EducationBenefitsClaimSerializer < ActiveModel::Serializer
-  attributes :id, :form, :regional_office, :confirmation_number
```

Comment:
@stevenjcumming How did you determine `id` wasn't needed anymore?

Reply:
With jsonapi-serializer the way to set the _id_ is through `set_id`. jsonapi-serializer tries `set_id :id` by default. 

`attribute :id` in AMS -> `set_id :id` in jsonapi-serializer (which is optional)

Also when I wrote the specs I made sure id wasn't at the attribute level and only the data level

Reply:
oh, okay 👍 thanks.